specification/netapp/resource-manager/Microsoft.NetApp/stable/2022-09-01/netapp.json

```diff
@@ -0,0 +1,8083 @@
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "A list of SubscriptionQuotaItems",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubscriptionQuotaItem"
+          }
+        }
+      }
+    },
+    "SubscriptionQuotaItem": {
+      "description": "Information regarding Subscription Quota Item.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "SubscriptionQuotaItem properties",
+          "$ref": "#/definitions/SubscriptionQuotaItemProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "SubscriptionQuotaItemProperties": {
+      "description": "SubscriptionQuotaItem Properties",
+      "type": "object",
+      "properties": {
+        "current": {
+          "description": "The current quota value.",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32"
+        },
+        "default": {
+          "description": "The default quota value.",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "netAppAccountList": {
+      "description": "List of NetApp account resources",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "Multiple NetApp accounts",
+          "items": {
+            "$ref": "#/definitions/netAppAccount"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of results."
+        }
+      }
+    },
+    "netAppAccount": {
+      "description": "NetApp account resource",
+      "type": "object",
+      "required": [
+        "location"
+      ],
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "properties": {
+          "description": "NetApp Account properties",
+          "$ref": "#/definitions/accountProperties",
+          "x-ms-client-flatten": true
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "The identity of the resource."
+        }
+      }
+    },
+    "Identity": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity"
+        }
+      ],
+      "properties": {
+        "userAssignedIdentities": {
```

Comment:
This is invalid usage, will cause breaking change failure, as the 'userAssignedIdentities' already exists in the 'resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity'. and you should use the common type directly

Reply:
updated